src/base/io/utilities/outputNetworkCytoscape.m

```diff
@@ -154,11 +154,11 @@
 fclose(fid);
 fclose(fidEdgeType);
 if (isfield(model,'subSystems'))
-    if (isfield(model,'subSystemsMet'))
-        for i = 1:length(model.mets)
-            fprintf(fidSubSys,'%s = %s\n',model.mets{i},model.subSystemsMet{i});
-        end
-    end
+%     if (isfield(model,'subSystemsMet'))
+%         for i = 1:length(model.mets)
+%             fprintf(fidSubSys,'%s = %s\n',model.mets{i},model.subSystemsMet{i});

```

Comment:
I removed it mainly because this is essentially only a printout where a field was used, that doesn't exist any more (and imo never existed in the first place)